backend/dataall/modules/s3_datasets_shares/services/s3_share_managed_policy_service.py

```diff
@@ -18,6 +28,9 @@
 EMPTY_STATEMENT_SID = 'EmptyStatement'
 
 S3_ALLOWED_ACTIONS = ['s3:List*', 's3:Describe*', 's3:GetObject']
+IAM_SERVICE_NAME = 'AWS Identity and Access Management (IAM)'
+IAM_SERVICE_QUOTA_NAME = 'Managed policies per role'
+DEFAULT_MAX_ATTACHABLE_MANAGED_POLICIES_ACCOUNT = 10
```

Comment:
If in the future we need to implement other Service Quota exceptions, then we will create something similar to the NamingConvention utility but for Service quotas where we defined the service names and quotas. But we don't need it at this moment, so having the base exceptions + listing the constants up here as constants is enough. Just keep it in mind for future implementations (and reviews ;) )